Section1_lab4_ShanHe_JoannaHuang_TiffanyJaya.Rmd

```diff
@@ -278,7 +280,113 @@ Although the assumption is not met, given the substantial sample size, we can be
+```
+Looking at the summary, every variable seems to be in an expected range except for the outlier 2177.1 in $wageser$. 
+
+```{r}
+# Look into wageser with an unusual max of 2177.1
+boxplot(data$wageser)
+```
+2177.1 is clearly an outlier in the data and could be possibly due to a measurement error. In this case, we decide not to include this $wageser$ to avoid confounding bias caused by potentially inaccurate data. 
+
+Moreover, let's look at how $west$ and $central$ distribute in the sample
+```{r}
+unique(cbind(data$west, data$central))
+```
+Note that although we don't have any counties that are both in west and central, as expected, we see some counties that neither in west or central. In order to consider the effect on different regions, we will need to use both indicator variables in our model. 
+
+Out of all the variables that aren't included in Model 1 or Model 2, we have decided to include:
+1) $west$ & $central$, as indicator variables, to control for the regional effect on crime rate
+2) $urban$, as an indicator variable, to control for the non-density impact of urbanization on crime crate
+3) $wagecon$, $wagetuc$, $wagetrd$, $wagefir$, $wageser$, $wagemfg$, $wagefed$, $wagesta$, $wageloc$ to control for the effects of wages in different industries have on our crime rate
+4) $pctmin$ and $ymale$ to control for the demographic effect on crime rate 
+
+Now let's look the relationship between the selected variables and our dependent variable $log(crime)$
+```{r}
+scatterplotMatrix(~ log(crime) + wagecon + wagetuc + wagetrd + wagefir, data = data)
+scatterplotMatrix(~ log(crime) + wagemfg + wagefed + wagesta + wageloc, data = data)
+scatterplotMatrix(~ log(crime) + pctmin + ymale, data = data)
+```
+
+None of the variables shows strong evidence of non-linear relationship with the dependent variable $log(crmrte)$. 
+
+We notice that the distributions for the variables $wagecon$, $wagetrd$, $wagefir$, $wagemfg$, $wagesta$, and $wageloc$ are positively skewed, since they are all postive values, we can apply log transformations to all of them. For the ease of the intepretation of the model, we apply log tranformation to the other wage related variable $wagetuc$ as well.
+
+Although $ymale$ is also positively skewed and could benefit from a log transformation in terms of normality, it's hard to interpret the slope parameter of its log transformation. Hence, we decided to leave it as it is. 
+
+Let's double check the linearity of the relationship between our transformed variables and our dependent variable $log(crime)$
+```{r}
+scatterplotMatrix(~ log(crime) + log(wagecon) + log(wagetuc) + log(wagetrd) + log(wagefir), data = data)
+scatterplotMatrix(~ log(crime) + log(wagemfg) + log(wagefed) + log(wagesta) + log(wageloc), data = data)
+scatterplotMatrix(~ log(crime) + pctmin + ymale, data = data)
+```
+
```

Comment:
Too many unnecessary scatterplotMatrix. I don't think we should explain linearity at this point. It's more for CLM 1.